marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -43,186 +43,152 @@
             dimensions: {
                 x: 0.3,
                 y: 0.3
             },
-            url: url,
+            imageURL: url,
             ignoreRayIntersection: false,
-            alpha: OVERLAY_SITTABLE_START_ALPHA,
+            alpha: SITTABLE_START_ALPHA,
             visible: true,
             emissive: true
-        });
+        },
+            "local"
+        );
 
         HMD.displayModeChanged.connect(sittableSwitchHMDToDesktopText);
-
         sittableLerpTransparency();
-
     }
 
-    // Fade's the sittable overlay over time
+    // Fade's the sittable local entity over time
     function sittableLerpTransparency() {
-
-        if (OVERLAY_DEBUG) {
+        if (UI_DEBUG) {
             print("sittableLerpTransparency");
         }
 
-        var startAlpha = OVERLAY_SITTABLE_START_ALPHA;
+        var startAlpha = SITTABLE_START_ALPHA;
         var changeAlpha = 0.01;
 
-        intervalSittableTransparency = Script.setInterval(function () {
-
-            startAlpha = startAlpha - changeAlpha; // My new alpha
-            Overlays.editOverlay(overlaySittable, { alpha: startAlpha });
+        // Update the alpha value on the sittable overlay
+        that.intervalSittableTransparency = Script.setInterval(function () {
+            startAlpha = startAlpha - changeAlpha;
+            Entities.editEntity(that.sittableID, { alpha: startAlpha });
 
-            if (startAlpha <= OVERLAY_SITTABLE_MIN_END_ALPHA) {
+            if (startAlpha <= SITTABLE_MIN_END_ALPHA) {
                 // Stop fading and keep overlay at the minimum alpha
-                Script.clearInterval(intervalSittableTransparency);
-                intervalSittableTransparency = null;
+                Script.clearInterval(that.intervalSittableTransparency);
+                that.intervalSittableTransparency = null;
             }
-
-        }, OVERLAY_SITTABLE_FADE_MS);
-
+        }, SITTABLE_FADE_MS);
     }
 
-    // Switch sittable overlay text from desktop mode "Click to Sit" to HMD mode "Trigger to Sit"
+    // Switch sittable local entity text from desktop mode "Click to Sit" to HMD mode "Trigger to Sit"
     function sittableSwitchHMDToDesktopText() {
-        if (overlaySittable) {
+        if (that.sittableID) {
             var url = HMD.active
-                ? OVERLAY_URL_SITTABLE_HMD
-                : OVERLAY_URL_SITTABLE_DESKTOP;
-
-            Overlays.editOverlay(overlaySittable, { url: url });
+                ? SITTABLE_IMAGE_URL_HMD
+                : SITTABLE_IMAGE_URL_DESKTOP;
+            Entities.editEntity(that.sittableID, { imageURL: url });
         }
     }
 
-    // Remove sittable overlay if it exists
+    // Remove sittable local entity if it exists
     function sittableRemove() {
-
-        if (OVERLAY_DEBUG) {
+        if (UI_DEBUG) {
             print("sittableRemove");
         }
 
-        if (overlaySittable) {
+        if (that.sittableID) {
```

Comment:
same comment as above here

Reply:
i'm seeing this pattern throughout the file - please address it! we can chat via Slack if you wish!